.travis.yml

```diff
@@ -1,7 +1,37 @@
 language: node_js
 node_js:
-- '0.10'
+- '6'
+before_install:
+- wget https://s3-us-west-2.amazonaws.com/cdn.thegrid.io/caliper/libvips/libccv-0.1.1.tar.gz
```

Comment:
So either
* Split out the things with custom Node.js C++ dependencies to another library
* Make the dependencies optional and handle them not existing
* Drop the offending NoFlo libraries from the build